src/main/java/edu/cornell/kfs/pmw/batch/PaymentWorksConstants.java

```diff
@@ -68,6 +76,13 @@
         public static final String PAYMENTWORKS_NEW_VENDOR_CREATE_ROUTE_ANNOTATION = "Generated by PaymentWorks New Vendor Create Batch Process";
     }
     
+    public static final class KFSPayeeAchMaintenaceDocumentConstants {
```

Comment:
There appears to be a misspelling in this classname. Sorry for not noticing it earlier.